qutebrowser/browser/webkit/webkitelem.py

```diff
@@ -111,6 +111,25 @@ def tag_name(self) -> str:
         self._check_vanished()
         return self._elem.tagName().lower()
 
+    def attr(self, name: str) -> str:
+        """Get 'name' attribute if exists"""
+        return self._js_dict.get('attributes', {}).get(name, '')
```

Comment:
I don't think webkit has an element for `_js_dict`. You should be able to copy `__getitem__` with a default argument here though